app/models/kuroko2/job_definition.rb

```diff
@@ -19,8 +19,6 @@ module PreventMultiStatus
     PreventMultiStatus::ERROR => [Kuroko2::Token::FAILURE, Kuroko2::Token::CRITICAL],
   }
 
-  self.locking_column = :version
```

Comment:
Why did you disable optimistic locking? http://api.rubyonrails.org/classes/ActiveRecord/Locking/Optimistic.html

Reply:
Because `version` column has removed.

https://github.com/cookpad/kuroko2/blob/74841d601abf9a0cb0b7191db9f3799f15dd4954/db/migrate/033_create_job_definition_revisions.rb#L3

Reply:
Why did you remove version column...?

Reply:
https://github.com/cookpad/kuroko2/pull/110#discussion_r180071515

Reply:
Oops...  I forgot that JobDefinition using optimistic locking. Please revert the commit 🙏

Reply:
I reverted the commit (4570e6056d7569bcd329e8f129873dc084b6d35d)